go.mod

```diff
@@ -39,31 +32,25 @@ require (
 	github.com/prometheus/common v0.7.0 // indirect
 	github.com/prometheus/procfs v0.0.5 // indirect
 	github.com/shurcooL/githubv4 v0.0.0-20191102174205-af46314aec7b // indirect
-	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/tektoncd/plumbing v0.0.0-20191216083742-847dcf196de9
 	go.opencensus.io v0.22.1
 	go.uber.org/atomic v1.4.0 // indirect
-	go.uber.org/zap v1.9.2-0.20180814183419-67bc79d13d15
-	golang.org/x/crypto v0.0.0-20191117063200-497ca9f6d64f // indirect
+	go.uber.org/zap v1.10.0
 	golang.org/x/net v0.0.0-20191119073136-fc4aabc6c914 // indirect
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e // indirect
 	golang.org/x/sys v0.0.0-20191119060738-e882bf8e40c2 // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	golang.org/x/tools v0.0.0-20191118222007-07fc4c7f2b98 // indirect
 	google.golang.org/api v0.10.0 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
-	google.golang.org/grpc v1.24.0 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.5 // indirect
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
-	k8s.io/client-go v0.0.0
-	k8s.io/code-generator v0.0.0
+	k8s.io/api v0.17.0
```

Comment:
Can't make `go` not changing thoses (`go mod tidy` & co always add this one…) It doesn't matter as they are replaced.. it's just a pain…

Reply:
So, we have to keep replacing them manually?

Reply:
For now I think so… I am hoping we won't, I need to try something out.. but I think until we bump to 1.15 (v0.15) or 1.17, we will still need the replace (but I might be wrong, I'll try that when I bump to 1.15 :angel: )